### ISSUE TITLE
Crash on Android

### ISSUE THREAD

**Issue by Numan-Munir** [leegeunhyeok/react-native-zendesk-messaging#24]
**Describe the bug**
I am trying to run react-native-zendesk-messaging on Android. It's works fine on iOS but issue on Android.

**To Reproduce**
1 - Open terminal
2 - Navigate to project 
3 - Run command yarn android

**Expected behavior**

**Environment **
 - Platform: [iOS, Android]
 - React Native Version [0.68.6]
 - Zendesk SDK Version [2.8.0]
 
 
 
 
 
 
 
<img width="1235" alt="Screenshot 2023-09-01 at 1 56 57 PM" src="https://github.com/leegeunhyeok/react-native-zendesk-messaging/assets/111237396/1e81b8b8-6f7b-489a-839c-08397c455b4a">
<img width="1224" alt="Screenshot 2023-09-01 at 1 57 14 PM" src="https://github.com/leegeunhyeok/react-native-zendesk-messaging/assets/111237396/42aa6c1b-a17d-4462-9541-b2a5a65fb2cc">

**Comment by leegeunhyeok** [comment]
Can I get logs that run with `--verbose` flag? @Numan-Munir

**Comment by Ankur-Float** [comment]
failing for me as well:

@leegeunhyeok here is the verbose logs:

FAILURE: Build failed with an exception.
```
* What went wrong:
Could not determine the dependencies of task ':app:processDebugResources'.
> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not find zendesk.messaging:messaging-android:2.13.0.
     Searched in the following locations:
       - https://oss.sonatype.org/content/repositories/snapshots/zendesk/messaging/messaging-android/2.13.0/messaging-android-2.13.0.pom
       - https://repo.maven.apache.org/maven2/zendesk/messaging/messaging-android/2.13.0/messaging-android-2.13.0.pom
       - file:/Users/ankurankur/Documents/Float/mobile-app/node_modules/jsc-android/dist/zendesk/messaging/messaging-android/2.13.0/messaging-android-2.13.0.pom
       - https://dl.google.com/dl/android/maven2/zendesk/messaging/messaging-android/2.13.0/messaging-android-2.13.0.pom
       - https://www.jitpack.io/zendesk/messaging/messaging-android/2.13.0/messaging-android-2.13.0.pom
     Required by:
         project :app > project :react-native-zendesk-messaging

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

**Comment by Ankur-Float** [comment]
i found a fix in latest react native version it does not allProjects section in android/build.gradle so instead of adding maven link in repositories directly I added it as below
```
allprojects {
    repositories {
     
        maven {
            url "https://zendesk.jfrog.io/artifactory/repo"
        }
     
    }
}
```
make sure this block of code is outside of buildscript block.

@leegeunhyeok I am not sure if this can be fixed in package itself

**Comment by leegeunhyeok** [comment]
@Ankur-Float This is already covered in the android section of getting started guide.
(In root `build.gradle`, add repository to allprojects)

https://github.com/leegeunhyeok/react-native-zendesk-messaging/blob/master/docs/getting-started.md#android

**Comment by pierricklete** [comment]
we faced a similar issue that was related to some conflicting build issue with Firebase performance https://github.com/firebase/firebase-android-sdk/issues/3155

we have fixed the issue by upgrading `AGP` to `v7.2` and `firebase:perf-plugin` to `v1.4.2`
